ordered_model/models.py

```diff
@@ -107,9 +107,15 @@ def bulk_create(self, objs, batch_size=None):
                     ] = self.filter_by_order_with_respect_to(obj).get_next_order()
                 setattr(obj, order_field_name, order_with_respect_to_mapping[key])
         else:
-            for order, obj in enumerate(objs, self.get_next_order()):
-                setattr(obj, order_field_name, order)
-        return super().bulk_create(objs, batch_size=batch_size)
+            next_order = self.get_next_order()
+            for obj in objs:
+                orig_order = getattr(obj, order_field_name)
```

Comment:
I think setting `order` explicitly should not be allowed at all. Users who want to make this more explicit in creating bulk objects with specific order should follow FIFO logic for their passed objects list

Reply:
It's a bit unconvinient if, for example, objects were dumped to json/yaml with order that differs from `order` field and then you try to load them back.

Reply:
I'm with @imomaliev on this one, you can sort the objects by order in your data loading, and add them in order.

Reply:
It breaks incapsulation - any (library) code that dumps/loads objects should check that object is instance of OrderedModel and handle such obejcts differently. Are you sure this is a better solution than make `bulk_create` just use the previously saved order if it's present?

Reply:
The dumping side of the code should be using the object's natural ordering, which for OrderedModel subclass is our sorted order, so loading should recover the desired order.

If we allow the order values to be user specified, and they are not *sorted anyway*, then during the load we will violate our ordering value constraints.